pyne/ensdf_processing.py

```diff
@@ -74,15 +74,49 @@ def alphad(inputdict_unchecked):
     proc.stdin.close()
     return inputdict_unchecked
 
+def delta(inputdict_unchecked):
+    """
+    This function calculates the best values of mixing ratios based of its analysis of
+    the angular correlation and conversion coefficient data.
+
+    Parameters
+    ----------
+    Input_dictionary : dictionary
+        dictionary that must have the following key-pair values:
+            input_file : input ensdf file
+            output_file : file for output to be written to (doesn't have to exist)
+
+    Notes
+    -----
+    Output Dictionary Values:
```

Comment:
Give the name of the return variable here, or just `rtn`.  Also give the variable type after the colon.